frontend/src/components/dialogs/SettingsAgentProfilesSection.css

```
@@ -0,0 +1,267 @@
+  inset: 0;
+  box-sizing: border-box;
+  visibility: hidden;
+  overflow: hidden;
+  border: 1px solid transparent;
+  padding: 8px;
+  font-family: var(--font-mono);
+  font-size: var(--font-size-sm);
+  line-height: 18px;
+  white-space: pre-wrap;
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)